public/dist/styles.css

```diff
@@ -10,4 +10,73 @@
o-action{background:#fff;text-align:center}.home-section.calls-to-action .section-header{color:#bc6e29}.footer{background:#aee0f6;text-align:center;padding:24px;color:#fff;font-size:14px}.footer .footer-container{max-width:960px;margin:0 auto;padding:0 10px}@media (min-width:490px){.footer .footer-container{display:flex;justify-content:space-between;align-items:center}}.footer .madewithlove{margin-bottom:10px}.footer .madewithlove a{color:hsla(0,0%,100%,.7)}@media (min-width:490px){.footer .madewithlove{text-align:right;margin-bottom:0}}.footer .heart{margin-bottom:10px;color:#fff;display:block;text-align:center;vertical-align:bottom}.footer .heart a{color:hsla(0,0%,100%,.7)}@media (min-width:490px){.footer .heart{text-align:right;margin-bottom:0}}.footer .heart i{color:#ff4d40;text-decoration:none}.footer .heart .fa{color:#fff;margin:0 3px;font-size:10px;animation:pound .35s infinite alternate;-webkit-animation:pound .35s infinite alternate}@-webkit-keyframes pound{to{transform:scale(1.4)}}@keyframes pound{to{transform:scale(1.4)}}@media (min-width:490px){.footer .contact-emails{text-align:right}}.footer .contact-emails a{color:#fff;display:inline-block;padding:0 4px}body.add-event{color:#282828}body.add-event,body.add-event .the-header{background:#fdecc2}body.add-event .the-header .right svg.burger-menu path{fill:#282828;stroke:#282828}body.add-event .container{max-width:760px}.add-event .intro{text-align:center;padding:72px 0}.add-event .intro h1,.add-event .intro p{margin:0}.add-event .intro h1{margin-bottom:12px}.add-event .intro p{font-size:1.2em}.add-event .AddEvent{display:flex;padding-bottom:96px}.add-event .AddEvent .form-wrap{flex:1 1 65%}.add-event .AddEvent .card-wrap{flex:1 1 35%}.add-event .AddEvent h3{color:#825329;margin:0;margin-bottom:24px}.add-event .card-wrap .sticky{padding-top:80px}.EventForm{background-color:#fdf2d3;margin-right:36px;border-radius:4px;box-shadow:0 2px 8px 0 rgba(0,0,0,.14),0 1px 4px -1px rgba(0,0,0,.2);padding:24px 32px}.EventForm .InputGroup{margin-bottom:28px}.EventForm .InputGroup label{color:#7b5521;display:block;font-weight:600;margin-bottom:8px}.EventForm .InputGroup input[type=email],.EventForm .InputGroup input[type=text]{border:none;background:none;border-bottom:2px solid #7b5521;display:inline-block;font-size:1em;padding:6px 0 4px}.EventForm .InputGroup ::placeholder{color:rgba(130,83,41,.3);font-weight:100}.EventForm .InputGroup.color input{text-transform:uppercase}.EventForm .InputGroup.color span{color:#7b5521;border-bottom:2px solid #7b5521;font-size:1em;padding:6px 4px 4px 0;display:inline-block}.EventForm .InputGroup .helper-text{display:block;color:rgba(130,83,41,.8);font-size:.9em;margin-top:8px}.EventForm .CheckboxInput{margin-bottom:12px}.EventForm .CheckboxInput .box{width:20px;height:20px;position:relative;margin-right:10px;border-radius:3px;display:inline-block;background:transparent;z-index:1;cursor:pointer;border:2px solid #825329;vertical-align:sub}.EventForm .CheckboxInput .inner
```

Comment:
Generally we don't want to modify the dist files, as they get built by Travis. Can you put these changes in one of the SASS files? Inside the `home.scss` file might be a good place.